packages/amplication-server/src/core/user/user.service.ts

```diff
@@ -7,12 +7,15 @@ import { UserRoleArgs } from "./dto";
 export class UserService {
   constructor(private readonly prisma: PrismaService) {}
 
-  findUser(args: Prisma.UserFindUniqueArgs): Promise<User> {
+  findUser(
+    args: Prisma.UserFindUniqueArgs,
+    includeDeleted = false
```

Comment:
Why did you set the default to false and then pass on each call to this function `true`?
Why didn't you just set the default to true? or just don't set a default?

Reply:
The current behaviour is to not include deleted resources, so this behaviour won't break any current logic